web/app/js/components/ServiceMesh.test.jsx

```diff
@@ -1,4 +1,5 @@
-import _ from 'lodash';
+import _cloneDeep from 'lodash/clonedeep';
```

Comment:
noticed this in ci, i think this need to be `lodash/cloneDeep`?

Reply:
oooooh good spot!